test/Extensions/TesterAdoNet/Tester.AdoNet.csproj

```diff
@@ -7,6 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="1.1.0" />
```

Comment:
I added these so one could quickly test also using these libraries by switching the hardcoded invariants. Not strictly necessary, although they're a dependency. I can remove if that's not OK.